Fabric.Authorization.AccessControl/src/app/modules/access-control/custom-group/custom-group.component.ts

```diff
@@ -593,4 +607,15 @@ export class CustomGroupComponent implements OnInit, OnDestroy {
 
     return user.subjectId;
   }
+
+  checkIfUserRemovedSelfFromGroup(currentUser: User, removedUsersList: IUser[]): boolean {
+    var matched = false;
+    removedUsersList.forEach(removedUser => {
+      if(removedUser.subjectId.toLowerCase == currentUser.profile.sub.toLowerCase) {
```

Comment:
I wonder if we store the user somewhere else. But it looks like you're getting the user direct from the auth service so maybe not

Reply:
I was trying to see if/where the user could be stored and couldn't find it. Doesn't mean we aren't though.